app/migrations/Version20181018125234.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20181018125234 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE committee_merge_histories ADD merged_by_id INT NOT NULL');
```

Comment:
normally it'll crash if the table already has some rows because of `NOT NULL`

Reply:
for some reason it's not crashing.
but i can't update a row that as such NULL value, indeed.

we plan to execute a SQL query after deploy so that existing lines has a merger by default